ls1intum/Artemis
# `Quiz exercises`: Improve layout for short answer matching options

### Checklist
#### General
#### Client

### Motivation and Context

Credit to https://github.com/ls1intum/Artemis/pull/5051

Close https://github.com/ls1intum/Artemis/issues/5000

It was "not clear that the two checkboxes and the slider affect the same underlying option about matching the answer text"
Also, it was not clear what the percent slider was used for.

### Description

We visually grouped the two checkboxes and the slider under a border, and placed a header "Answer Matching" to make it more obvious that these options are part of the same setting.
We also added a German translation string for the "Answer Matching" string.

### Steps for Testing
Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Navigate to Course Management
3. Click on Exercises
4. Click on Create Quiz
5. Click on "Add Short-Answer Question"
6. Check that the Answer Matching div looks as shown in the screenshots and works correctly

### Review Progress

#### Code Review
#### Manual Tests

### Screenshots

#### Before


![image](https://user-images.githubusercontent.com/11006274/179283631-62d00127-2527-4f3c-b462-8758d5b40cc8.png)

#### After

English UI:

![image](https://user-images.githubusercontent.com/11006274/179283647-42e923c4-9b65-44f8-a7e0-b52f8abec312.png)


German UI:

![image](https://user-images.githubusercontent.com/11006274/179283663-83f2e852-2a43-4e09-92a9-3459ea2f57bd.png)
